crates/sui-source-validation-service/src/main.rs

```diff
@@ -21,6 +21,7 @@ pub async fn main() -> anyhow::Result<()> {
     let package_config = parse_config(args.config_path)?;
     let sui_config = sui_config_dir()?.join(SUI_CLIENT_CONFIG);
     let context = WalletContext::new(&sui_config, None, None).await?;
-    initialize(&context, &package_config).await?;
+    let tmp_dir = tempfile::tempdir()?;
```

Comment:
The server always clones to a temporary dir for now. I will lift this and other bits to environment variables in follow up PRs.